grype/match/ignore.go

```diff
@@ -28,14 +28,15 @@ type IgnoredMatch struct {
 // specified criteria must be met by the vulnerability match in order for the
 // rule to apply.
 type IgnoreRule struct {
-	Vulnerability    string            `yaml:"vulnerability" json:"vulnerability" mapstructure:"vulnerability"`
-	Reason           string            `yaml:"reason" json:"reason" mapstructure:"reason"`
-	Namespace        string            `yaml:"namespace" json:"namespace" mapstructure:"namespace"`
-	FixState         string            `yaml:"fix-state" json:"fix-state" mapstructure:"fix-state"`
-	Package          IgnoreRulePackage `yaml:"package" json:"package" mapstructure:"package"`
-	VexStatus        string            `yaml:"vex-status" json:"vex-status" mapstructure:"vex-status"`
-	VexJustification string            `yaml:"vex-justification" json:"vex-justification" mapstructure:"vex-justification"`
-	MatchType        Type              `yaml:"match-type" json:"match-type" mapstructure:"match-type"`
+	Vulnerability        string            `yaml:"vulnerability" json:"vulnerability" mapstructure:"vulnerability"`
+	RelatedVulnerability string            `yaml:"related-vulnerability,omitempty" json:"related-vulnerability,omitempty" mapstructure:"related-vulnerability"`
```

Comment:
Only change here is adding `RelatedVulnerability`

Reply:
Should we fill `Vulnerability` here and change `RelatedVulnerability` into a boolean (and probably rename to `IncludeAliases` or similar)? 

This way there aren't two places to put the same information with slightly different semantics, it would be a new field with no overlapping use cases.